k8s-install/scripts/install-cni.sh

```diff
@@ -144,11 +144,13 @@ EOF
 fi
 
 
+CNI_MTU=${CNI_MTU:-1500}
 # Insert any of the supported "auto" parameters.
 grep "__KUBERNETES_SERVICE_HOST__" $TMP_CONF && sed -i s/__KUBERNETES_SERVICE_HOST__/${KUBERNETES_SERVICE_HOST}/g $TMP_CONF
 grep "__KUBERNETES_SERVICE_PORT__" $TMP_CONF && sed -i s/__KUBERNETES_SERVICE_PORT__/${KUBERNETES_SERVICE_PORT}/g $TMP_CONF
 sed -i s/__KUBERNETES_NODE_NAME__/${KUBERNETES_NODE_NAME:-$(hostname)}/g $TMP_CONF
 sed -i s/__KUBECONFIG_FILENAME__/calico-kubeconfig/g $TMP_CONF
+grep "__CNI_MTU__" $TMP_CONF && sed -i s/__CNI_MTU__/${CNI_MTU}/g $TMP_CONF
```

Comment:
You could do this in one line (without the defaulting above):

```
grep "__CNI_MTU__" $TMP_CONF && sed -i s/__CNI_MTU__/${CNI_MTU:-1500}/g $TMP_CONF
```

You can also drop the `grep` part from the front - that was required for the `KUBERNETES_` variables because on non-k8s systems they don't exist.

So just:
```
sed -i s/__CNI_MTU__/${CNI_MTU:-1500}/g $TMP_CONF
```

Reply:
Oh I thought someone can just have "mtu" : 1100 directly in cni_config in which case __CNI_MTU__ won't be there. So included grep part to take care of that case. Won't that happen?